src/main/java/org/update4j/service/DefaultBootstrap.java

```diff
@@ -130,8 +129,16 @@ void parseArgs(List<String> args){
             if (m.matches()) {
                 validateNotSet(cert, "cert");
                 cert = m.group(1);
+                try{
+                	CertificateBuilder.from(cert);
+				} catch(Exception e) {
+					throw new RuntimeException("Certificate option specified, but certificate doesn't exist: "+cert,e);
+				}
                 continue;
             }
+
+            //unknown argument
+			throw new IllegalArgumentException("Unknown argument: "+arg);
```

Comment:
Since args are passed over to business app (via main or launcher) I wouldn't blow up everything for an unknown argument.